styleguide/_themes/global/scss/modifiers.scss

```diff
@@ -40,6 +40,26 @@ img { max-width: 100%; }
   padding: 40px 0 60px;
 }
 
+//equal height columns
+.row-eq-height {
+  display: -webkit-flex;
+  display: -ms-flexbox;
```

Comment:
After you rebate I think there's a mixin for flexboxes now. Check the vendor prefix file to confirm but we should have that as of last Friday.